docs/scenarios/Build-your-own-scenario.md

```diff
@@ -0,0 +1,130 @@
+# Build your own scenario
+
+**Inductiva API** allows users to build their own scenarios and run simulations on them. In this way, users will be able to iterate over their parameters of choice as they wish, without needing to create input files for each of them. The main steps to build a scenario are:
+
+- Template your input files with your parameters of choice.
+- Create your scenario class that inherits from `inductiva.scenarios.Scenario`.
+- Run your simulations.
+
+## Template your input files
```

Comment:
This is fine. I don't think we need to introduce scenarios for now. Let's just talk about templates. Or, let's actually have a section on "generalizing your simulation" or "running variations of your simulation". I think if we can avoid the word scenario for now, it's great.